drivers/hmis/app/graphql/types/hmis_schema/has_enrollments.rb

```diff
@@ -33,9 +35,11 @@ def resolve_enrollments(scope = object.enrollments, **args)
 
       private
 
-      def scoped_enrollments(scope, sort_order: :most_recent, include_in_progress: false)
+      def scoped_enrollments(scope, sort_order: :most_recent, include_in_progress: false, project_types: nil, client_search_term: nil)
         scope = scope.viewable_by(current_user)
         scope = scope.where.not(project_id: nil) unless include_in_progress
+        scope = scope.with_project_type(project_types) if project_types.present?
+        scope = scope.joins(:client).merge(Hmis::Hud::Client.matching_search_term(client_search_term)) if client_search_term.present?
```

Comment:
Is it bad to join client here, since it's modifying the result thats returned?

Reply:
I don't think it's a problem.  It does mean if you have a client search term, you'll be limited to enrollments with a client, but I'd think you'd always want that, and should always have a client.  If it bothers you, you could just join client on line 39 so it always happens.

Reply:
Ok. The GraphQL schema requires enrollment to have `client`, so if that were to happen it would already be throwing a 500.